### PR TITLE
docs: More doc updates

### PR DESCRIPTION
**Pull request by laf**
#### Please note

> Please read this information carefully. You can run `./scripts/pre-commit.php` to check your code before submitting.

- [x] Have you signed the [Contributors agreement](http://docs.librenms.org/General/Contributing/)
- [x] Have you followed our [code guidelines?](http://docs.librenms.org/Developing/Code-Guidelines/)

### CODE REVIEW

File: doc/Extensions/RRDCached.md [hunk 1]
```diff
@@ -11,6 +11,23 @@ To enable this set the following config:
...
 
+### Support matrix
+
+Shared FS: Is a shared filesystem required?
+
+Features: Supported features in the version indicated.
+
+          G = Graphs.
+
+          C = Create RRD Files.
```
Comment (murrant) [inline doc/Extensions/RRDCached.md]: Put tune and update in here? 

Also the table below could be:
| Version | Shared FS Required | With Shared FS | Without Shared FS| 
| ------- | :-------: | :-------: | -------- |
| **<1.5.5**   | Yes | C,U,G,T  |        |
| **>=1.5.5** | No | C,U,G,T  | C,U  |

File: doc/Developing/Creating-Documentation.md [hunk 2]
```diff
@@ -3,16 +3,35 @@ source: Developing/Creating-Documentation.md
...
+
+  - pymdownx.tasklist
+  - mdx_del_ins
+
+This means you can use:
+
+  - `~~strikethrough~~` to perform ~~strikethrough~~
+  - [X] `- [X] List items` 
+  - Url's can be made `[like this](http://www.librenms.org)` [like this](http://www.librenms.org)
+  - Code can be placed in `` for single line or ``` for multiline.
```
Comment (murrant) [inline doc/Developing/Creating-Documentation.md]: I think you need to escape the `s here

File: doc/Extensions/RRDCached.md (hunk 1 above)
Comment (laf) [inline doc/Extensions/RRDCached.md]: But tune doesn't work over rrdcached by all accounts.

I'll add update in.

File: doc/Developing/Creating-Documentation.md (hunk 2 above)
Comment (laf) [inline doc/Developing/Creating-Documentation.md]: Tested it by using mkdocs serve and it's fine there.

**Comment by laf** [comment]
Updated

**Pull request merged**